source/_meta/_00-head.mustache

```diff
@@ -5,7 +5,6 @@
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width" />
 
-    <link rel="stylesheet" href="../../../../css/style.css?{{ cacheBuster }}" media="all" />
```

Comment:
This isn't our styles.css file -- it's a file from pattern lab core.  I think we should leave this in.